rdr_service/workflow_management/ppsc/ppsc_intake_to_ps_queries.py

```diff
@@ -368,21 +367,29 @@ def get_survey_completion_activity_to_stream(project: str,
   and data_element_name IN ("activity_status", '​activity_status', "gender_genderidentity","biologicalsexatbirth_sexatbirth","thebasics_sexualorientation","race_whatraceethnicity","educationlevel_highestgrade","income_annualincome")
   and se.ignore_flag = 0
 )
+, aian_flag AS (
+  SELECT
+    participant_id,
+    MAX(CASE
+      WHEN data_element_name = 'race_whatraceethnicity'
+           AND data_element_value = 'WhatRaceEthnicity_AIAN'
+      THEN 1 ELSE 0 END) AS is_aian
+  FROM `{project}.{source_dataset}.ppsc_survey_completion_event`
+  WHERE ignore_flag = 0
+  GROUP BY participant_id
+)
```

Comment:
@j-kanuch, maybe I'm reading this wrong. But would this flag a participant as AIAN if they've _ever_ provided that as an answer? If so, should their AIAN status be unset if they ever answer that they're not?

Reply:
@kenny-skaggs Yes, you are reading this correctly. I'll need to follow up on what the requirement is, whether we should be updating this if they remove the AIAN answer from their list of race/ethnicity answers.